magda-int-test-ts/src/ServiceRunner.ts

```diff
@@ -86,6 +86,8 @@ export default class ServiceRunner {
     // in order to setup the indices in search engine.
     // however, indexer will auto exit if this field is set to false.
     public enableIndexer = false;
+    public hybridSearchVectorWorkloadMode: "on_disk" | "in_memory" =
```

Comment:
Good job on injecting new config to make searchApi support hybrid search 💯 
As the ServiceRunner.ts is a common utility to run test cases I guess we might need to update searchApi config for other tests, is it possible to replace the specific `hybridSearchVectorWorkloadMode` config property here with a more generic config property e.g.:
```typescript
public searchApiConfig: string | null = null
```
Here, we only inject / set `-Dconfig.file=${confFilePath}` when `searchApiConfig` is a non-empty string.
This way, any test cases can pass in a new config content as string to get a customised search API for different purpose.